examples/guestbook/frontend-service.json

```diff
@@ -17,6 +17,7 @@
       ],
       "selector":{
          "name":"frontend"
-      }
+      },
+      "createExternalLoadBalancer": true
```

Comment:
Should we have a separate config for the e2e? I realize that sounds a bit silly, but it seems like our example goals are a bit different from the e2e ones.